Gemfile.lock

```diff
@@ -20,25 +20,25 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (5.2.8.1)
-      actionpack (= 5.2.8.1)
+    actioncable (5.2.8)
+      actionpack (= 5.2.8)
```

Comment:
**question:** Are these downgrades from Rails 5.2.8.1 to 5.2.8 intentional here?

Reply:
Oops, apparently I didn't properly resolve the merge conflicts.